omod/src/main/resources/config.xml

```diff
@@ -18,7 +18,7 @@
 	
 	<!-- Required Modules -->
 	<require_modules>
-
+		<require_module version="${uicommonsVersion}">org.openmrs.module.uicommons</require_module>
```

Comment:
Why do you require the uicommons module?

Reply:
We need uiframework for now in the fragement controller,So I changed to uiframework